content/maintainers-guide/project-organization.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Project organization
+weight: 1
+---
+
+# Project organization
+
+## Project roles
+
+### Product Manager
+
+Product Managers provide the project's product vision, ensuring that the software provides an answer to the needs of its different types of users.
+
+Current product managers:
+
+- Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon))
```

Comment:
Maybe we could clarify that @fabienvallon is the lead PM ?

Reply:
I disagree. Why ? In the open source project, PM have the same status like maintainers. No lead. Like @eternoendless in maintainers group.

Reply:
Same. Until we determine a proper role for Fabien and me, officially we're equal.

Reply:
I agree Progi1984.